src/win32.mak

```diff
@@ -93,6 +96,8 @@ TOOLS_DIR=..\..\tools
 CC=dmc
 # D compiler (set with env variable)
 #HOST_DC=dmd
+# For error detection:
+HOST_DC=Please_define_the__HOST_DC__environment_variable
```

Comment:
This works to display a nicer error, but it no longer allows setting HOST_DC in the environment. It is only overwritten on the command line.

Reply:
On wine at least it works with HOST_DC set in the environment.

Reply:
Could you try it on native Windows? Thx!

Reply:
That's what I did. Are you sure to invoke DigitalMars make?

An explicit test of the variables works, too, but has to be added as a dependency to the defaulttarget and maybe others:
```
checkvars:
	cmd /c if "$(HOST_DC)" == "" (echo Please set HOST_DC && exit 1)
```
(DM make does not seem to interpret the result of executing an implicit shell command, so I had to make it explicit).

Reply:
Interesting, I'll look into this. For what it's worth I use `wine make -f win32.mak` that I assume uses dmd make, but I'll check when I get back to that machine.

Reply:
BTW we could also set the `DC_HOST` variable to `dmd` which is the default that's used for `posix.mak` and then simply add a nice error message as @rainers suggested.

(if DM make supports this)

Reply:
I tried to get this "default" functionality working with DM make and failed...

Reply:
Nevermind, got it it to work (https://github.com/dlang/dmd/pull/7468).